aws/cloudformation/standalone/gen_ai_curriculum/monitoring/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "scripts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Optional: instead of having to first npm install in the `monitoring` directory and then go into the specific alarms or dashboard directory, we could add some convenience scripts here to do the second part for you, like
```
"scripts": {
  "alarms": "cd ./alarms && npx ts-node createAlarms.ts",
  ...
}
```
So you just have to run `npm install && npm run alarms` from this directory. Not required though, going into the specific directories is fine especially if the commands do end up diverging more.

Reply:
Thanks Sanchit! I think I'll leave as-is for now.